src/base/bittorrent/session.cpp

```diff
@@ -282,6 +301,9 @@ Session::Session(QObject *parent)
     , m_maxUploadsPerTorrent(BITTORRENT_SESSION_KEY("MaxUploadsPerTorrent"), -1, lowerLimited(0, -1))
     , m_isUTPEnabled(BITTORRENT_SESSION_KEY("uTPEnabled"), true)
     , m_isUTPRateLimited(BITTORRENT_SESSION_KEY("uTPRateLimited"), true)
+    , m_utpMixedMode(BITTORRENT_SESSION_KEY("uTPMixedMode"), static_cast<BitTorrent::MixedModeAlgorithm>(m_isUTPEnabled.value())
```

Comment:
>Do we have dependent default value here?

Ah, damn it, I didn't realize until now... it doesn't need to depend on others.
Changed.